tests/e2e/createcollection.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { expect, test } from '@playwright/test'
+
+const chains = ['Kusama', 'RMRK2', 'KusamaHub [Beta]', 'PolkadotHub [Beta]']
+
+test('Collection Creation', async ({ page }) => {
+  let i = 0
+  //E2E connection
+  await page.goto('/e2e-login')
+  await page.waitForTimeout(10000)
+  await expect(page.getByTestId('mockAddress')).toHaveText('true')
+  await page.goto('/create/collection')
+  //loop trough possible chains
+  for (const data of chains) {
+    await page.getByTestId('collection-chain').selectOption(`${chains[i]}`)
+    i++
+    //upload image logo
+    await page.setInputFiles(
+      'input[type="file"]',
+      'tests/e2e/unsplash-image.jpg'
+    )
+    //checks if Collection Name Title Exists
+    await expect(
+      page
+        .getByTestId('collection-name')
+        .filter({ has: page.getByText('Collection Name') })
+    ).toHaveCount(1)
+    // defines a variable containing the collection name data-testid
+    const collectionName = page.getByTestId('collection-name')
+    // Fill and check if Collection Name works properly
+    await collectionName.locator('input').fill('Kodadot Automatic Collection')
+    await collectionName.locator('input').fill('')
```

Comment:
2 times filled?

Reply:
filled with text to see if the field works and then filled with nothing and clicked elsewhere in order to trigger warning on field, should i do it in another way?

Reply:
I would create separete test for these thing something like 
```
test('Show warning message on field when collection name empty, {....})
```

Reply:
hmmmm, ok!